pkgs/development/haskell-modules/patches/hie-testsuite.patch

```diff
@@ -0,0 +1,40 @@
+diff --git a/test/HaRePluginSpec.hs b/test/HaRePluginSpec.hs
```

Comment:
Ditto. Can this file be downloaded from somewhere else?

Reply:
This is a workaround, not a real fix, and doesn't exist anywhere yet.